hazelcast/pom.xml

```diff
@@ -357,7 +347,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-resources-plugin</artifactId>
-                <version>3.2.0</version>
+                <version>${maven.resources.plugin.version}</version>
```

Comment:
Please remove the version specification. It's handled by the `pluginManagement` in the `hazelcast-root`.